Sources/VLCDocumentClass.swift

```diff
@@ -0,0 +1,21 @@
+/*****************************************************************************
+ * VLCDocumentClass.swift
+ * VLC for iOS
+ *****************************************************************************
+ * Copyright (c) 2018 VideoLAN. All rights reserved.
+ * $Id$
+ *
+ * Authors: Carola Nitz <nitz.carola # googlemail.com>
+ *
+ * Refer to the COPYING file of the official project for license.
+ *****************************************************************************/
+
+import Foundation
+
+@objc(VLCDocumentClass)
+class DocumentClass: UIDocument {
```

Comment:
Do we need this class or was it to prepare for future checks?

Reply:
we need to have a subclass as mentioned in the Documentation. You can't use it directly

Reply:
I see, thanks for the explanation!